js/customize-snapshots.js

```diff
@@ -290,14 +474,15 @@
 			options
 		);
 
+		data = _.extend( args, {
+			nonce: api.settings.nonce.snapshot,
+			customize_snapshot_uuid: component.data.uuid
+		} );
+
 		data = _.extend(
 			{},
 			api.previewer.query(),
-			{
-				nonce: api.settings.nonce.snapshot,
-				customize_snapshot_uuid: component.data.uuid,
-				status: args.status
```

Comment:
Why do we need to extend args? Are the args being used in the preview query for something else now?